amoro-ams/src/main/java/org/apache/amoro/server/optimizing/maintainer/TableMaintainer.java

```diff
@@ -60,9 +60,9 @@ default void cleanDanglingDeleteFiles(TableRuntime tableRuntime) {
 
   static TableMaintainer ofTable(AmoroTable<?> amoroTable) {
     TableFormat format = amoroTable.format();
-    if (format == TableFormat.MIXED_HIVE || format == TableFormat.MIXED_ICEBERG) {
+    if (TableFormat.MIXED_HIVE.equals(format) || TableFormat.MIXED_ICEBERG.equals(format)) {
```

Comment:
How about replacing with TableFormat#in?

Reply:
fixed